README.md

```diff
@@ -28,10 +29,12 @@ npm install seneca-entity
 ```
 
 ## Using Seneca Entity
+The code below shows how to load this plugin as well as an in memory store.
 
 ```js
 require('seneca')()
   .use('entity')
+  .use('mem-store')
```

Comment:
@mihaidma Yes. I wonder, should we load it by default in this module? All of the store stuff is now out of Seneca.